### PR TITLE
oo/get members api

### PR DESCRIPTION
**Pull request by OnyebuchiO**

### CODE REVIEW

File: app/memorial/controllers/display.py [hunk 1]
```diff
@@ -48,3 +49,58 @@ def display_single_memorial_for_user(user_id, memorial_id):
...
+    for role_type in role_docs:
+        user_roles.append(str(role_type.role))
+
+    # Creates a JSON response displayed when api route is called
+    for mem in user_doc:
+        members["email"] = mem.email
+        members["firstName"] = mem.first_name
+        members["id"] = str(mem.id)
+        members["lastName"] = mem.last_name
+        members["role"] = user_roles.pop(0)
```
Comment (amanzanero) [inline app/memorial/controllers/display.py:99]: image

i made these changes and it returned all of them

Comment (amanzanero) [reply app/memorial/controllers/display.py:99]: locally, i didn't commit

File: app/memorial/routes.py [hunk 2]
```diff
@@ -83,3 +83,12 @@ def settings(memorial_id):
         return {"msg": response}, code
 
     return {"memorial": response}, 200
+
+
+@memorial_bp.route('/members/<memorial_id>', methods=['GET'])
```
Comment (amanzanero) [inline app/memorial/routes.py:88]: the route should be `/<memorial_id>/members` here

File: app/memorial/routes.py [hunk 3]
```diff
@@ -83,3 +83,12 @@ def settings(memorial_id):
...
+@jwt_required(optional=True)
```
Comment (amanzanero) [inline app/memorial/routes.py:89]: this shouldn't be optional here
```suggestion
@jwt_required()
```

**Review by OnyebuchiO — COMMENTED** [review]
Now shows all users associated

**Review by amanzanero — APPROVED** [review]
beautiful

**Pull request merged**